Sources/Public/Internal/SignalContext.swift

```diff
@@ -3,6 +3,10 @@ import Foundation
 protocol SignalContext: CustomStringConvertible {
     var allAttributes: Attributes { get }
     var attributes: Attributes { get set }
+    // File attachments are stored to disk as URLs
+    var attachments: Attachments { get set }
+    // File attachments are used in `BacktraceReport` as string paths
+    var attachmentPathsArray: [String] { get }
```

Comment:
What's the reason for storing `attachments` and `attachmentPathsArray`? Also, I suggest renaming `attachementPathsArray` to `attachmentPaths`.

Reply:
We need `attachmentPathsArray` for live reports. I tagged you where we use `attachmentPathsArray` in `BacktraceReporter`. 

`attachmentPaths` is definitely a better name, will rename.